mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td

```diff
@@ -1566,7 +1566,7 @@ def Tosa_PadOp : Tosa_InferShapedTypeOp<"pad"> {
 
   let arguments = (ins
     Tosa_RankedTensor:$input1,
-    Tosa_Int32Or64Tensor:$padding,
+    2DTensorOf<[Tosa_Int32Or64]>:$padding,
```

Comment:
Thought that `padding` is of rank 1 and dimensions size equal to `2 * rank(input)`.

Reply:
The Spec says the type is `shape_t<[2*rank(shape1)]>`, I'm not sure it's 1D or 2D. But I find that the padding of `tosa.pad` before is all 2D.